lua/fzf-lua-frecency/init.lua

```diff
@@ -17,67 +17,82 @@ local M = {}
 
 
 local function get_files_cmd(opts)
+  -- https://github.com/ibhagwan/fzf-lua/blob/e40e2337611fa426b8bcb6989fc310035c6ec4aa/README.md?plain=1#L831-L833
+  local default_fd_opts = [[--color=never --hidden --type f --type l --exclude .git]]
+  local default_find_opts = [[-type f \! -path '*/.git/*']]
+  local default_rg_opts = [[--color=never --hidden --files -g "!.git"]]
+
+  local fd_opts = h.default(opts.fd_opts, default_fd_opts)
+  local find_opts = h.default(opts.find_opts, default_find_opts)
+  local rg_opts = h.default(opts.rg_opts, default_rg_opts)
+
   local cmd
-  local fd_args = table.concat({
-    "--absolute-path",
-    "--type", "f",
-    "--type", "l",
-    "--exclude", ".git",
-  }, " ")
-  if vim.fn.executable "fdfind" == 1 then
-    cmd = string.format("fdfind %s", fd_args)
-  elseif vim.fn.executable "fd" == 1 then
-    cmd = string.format("fd %s", fd_args)
-  elseif vim.fn.executable "rg" == 1 then
-    -- return [[rg --files -g "\!.git" "$(pwd)"]]
-    cmd = string.format([[rg --files -g "!.git"]], opts.cmd or vim.uv.cwd())
+  if vim.fn.executable "fdfind" == h.vimscript_true then
+    cmd = ("fdfind %s"):format(fd_opts)
+  elseif vim.fn.executable "fd" == h.vimscript_true then
+    cmd = ("fd %s"):format(fd_opts)
+  elseif vim.fn.executable "rg" == h.vimscript_true then
+    cmd = ("rg %s"):format(rg_opts)
+  elseif vim.fn.executable "find" == h.vimscript_true then
+    cmd = ("find %s"):format(find_opts)
   else
-    FzfLua.utils.warn "[fzf-lua-frecency] 'all_files' requires 'fd' or 'rg'."
+    FzfLua.utils.warn "[fzf-lua-frecency] 'all_files' requires 'fd', 'rg', or 'find'."
     return nil
   end
-  for k, v in pairs {
-    follow = opts.toggle_follow_flag or "-L",
-    hidden = opts.toggle_hidden_flag or "--hidden",
-    no_ignore = opts.toggle_ignore_flag or "--no-ignore",
-  } do
+
+  local toggle_flags = {
+    follow = h.default(opts.toggle_follow_flag, "-L"),
+    hidden = h.default(opts.toggle_hidden_flag, "--hidden"),
+    no_ignore = h.default(opts.toggle_ignore_flag, "--no-ignore"),
+  }
+
+  for flag_name, flag_value in pairs(toggle_flags) do
```

Comment:
This section took me a bit to wrap my head around :sweat_smile: I renamed some vars but didn't change any logic